src/commands/force/user/create.ts

```diff
@@ -36,6 +37,36 @@ interface FailureMsg {
   message: string;
 }
 
+const permsetsStringToArray = (fieldsPermsets: string | string[]): string[] => {
+  if (!fieldsPermsets) return [];
+  return isArray<string>(fieldsPermsets)
+    ? fieldsPermsets
+    : fieldsPermsets.split(',').map((item) => item.replace("'", '').trim());
```

Comment:
@mshanemc item.replace assumes the items are wrapped in single quotes. Any chance item could be wrapped in double quotes?

Reply:
I tried to to escape the quotes on the comment above, but github seems to be parsing them out.  I meant `arg="\"something\""`

Reply:
How about arg='"something"'?